### ISSUE TITLE
Got warning: Couldn't fetch mysqli_result in ...

### ISSUE THREAD

**Issue by codearryaas** [pressjitsu/wpdb-phpsessions#1]
Warning: mysqli_free_result(): Couldn't fetch mysqli_result in ..\wp-includes\wp-db.php on line 1425.

**Comment by soulseekah** [comment]
Thanks for your report. Needs more info, though, what page were you accessing? What command was running before the warning?

**Comment by codearryaas** [comment]
Warning is shown in every page. No command was running. I checked in Xampp. 
If I remove clone from https://github.com/pressjitsu/wpdb-phpsessions/blob/master/wpdb-phpsessions.php#L142 then it doesn't show any errors.

**Comment by soulseekah** [comment]
Pretty weird. What version of WordPress are you using? If 4.4.1 latest:

Can you please add `if ( ! $this->result ) debug_print_backtrace();` on line 1425 of wp-db.php for debugging purposes, before the call to mysqli_free_result.

**Comment by codearryaas** [comment]
Yes I am using WordPress version: 4.4.1.
I added `if ( ! $this->result ) debug_print_backtrace();` on line 1425 of wp-db.php but output was same warning no other results displayed.

**Comment by soulseekah** [comment]
If you disable the plugin is the warning gone?

**Comment by codearryaas** [comment]
Yes if I disable then the warning is gone.
I am running PHP version: 5.6.12.

**Comment by soulseekah** [comment]
How did you enable the plugin in the first place? It requires extra configuration to enable it.

**Comment by codearryaas** [comment]
I changed following:

```
self::$config = self::maybe_user_config( array(
            'enable' => true,
        ) );
```

Sorry what are the extra configurations?

**Comment by soulseekah** [comment]
Yep, that's fine. But we're still not getting any warnings here, so hard to tell what might be wrong, we'll investigate a bit more and see why clone would generate a warning there. Thanks.

**Comment by codearryaas** [comment]
Ok thanks, let me know if you got any solutions. I'll also check from my side.

**Comment by eriktelepovsky** [comment]
Hi, same here. Is removing $instance->wpdb a good solution? Can I replace it by global $wpdb?

**Comment by soulseekah** [comment]
@eriktelepovsky are you on Windows, too?

**Comment by eriktelepovsky** [comment]
Nope, Mac OX 10.11.4 and the issue is also on my production server (Debian).

**Comment by ben-neale** [comment]
Did you ever resolve this. I have the same issue. It was working fine this morning.

ETA: Please ignore it was a completely different issue causing my problem, this was just a byproduct of my dodgy code!

**Comment by tomjn** [comment]
@atticus7 what was the cause/solution in your case?
